src/extra.c

```diff
@@ -366,8 +366,10 @@ static int extra_libc(smite_state *S)
             char *s2 = (char *)smite_native_address_of_range(S, str2, 0);
             if (s1 == NULL || s2 == NULL)
                 error = -9;
-            if (error == 0)
-                PUSH(rename(s2, s1));
+            if (error == 0) {
+                int res = rename(s2, s1);
```

Comment:
Given the guard `error == 0`, why is the intermediate variable `res` necessary?

Reply:
It's not, you're right.

Reply:
Great, can you update the PR accordingly please?

Reply:
I think this has already been obviated by my next pull request (branch issue35) which touched the same code.